frontend/src/pages/notebookController/const.ts

```diff
@@ -1,5 +1,6 @@
 export const CUSTOM_VARIABLE = 'Custom variable';
 export const EMPTY_KEY = '---NO KEY---';
+export const MOUNT_PATH = '/opt/app-root/src';
```

Comment:
This is the same route as the [backend notebook workingDir](https://github.com/opendatahub-io/odh-dashboard/blob/main/backend/src/routes/api/notebooks/notebookUtils.ts#L79) right? Should we make some sorta comment / constant-ize the backend variable as well?

The idea here is keep them sync, no?

Reply:
Yes, good idea